src/plugins/wizard/public/saved_visualizations/saved_visualization_references.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { SavedObjectReference } from '../../../../core/public';
+import { WizardVisSavedObject } from '../types';
+import { injectSearchSourceReferences } from '../../../data/public';
+
+export function injectReferences(
+  savedObject: WizardVisSavedObject,
+  references: SavedObjectReference[]
+) {
+  if (savedObject.searchSourceFields) {
```

Comment:
what's the purpose of this check?

Reply:
We can probably discard the check, but I was following the format of injectReferences in the visualization plugin here: [InjectReferences](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/visualizations/public/saved_visualizations/saved_visualization_references.ts#L97)